src/prefect/environments/storage/docker.py

```diff
@@ -79,19 +72,50 @@ def __init__(
         self.image_name = image_name
         self.image_tag = image_tag
         self.python_dependencies = python_dependencies or []
+        self.python_dependencies.append("wheel")
+
         self.env_vars = env_vars or {}
+        self.env_vars["PREFECT__USER_CONFIG_PATH"] = "/root/.prefect/config.toml"
+
         self.files = files or {}
         self.flows = dict()  # type: Dict[str, str]
         self._flows = dict()  # type: Dict[str, "prefect.core.flow.Flow"]
         self.base_url = base_url or default_url
         self.local_image = local_image
+        self.extra_commands = []
```

Comment:
This feels like a really convenient abstraction - I'd love let's expose this as a new keyword argument for `Docker` storage.

Reply:
Maybe, but the ultimate abstraction to this is the `base_image` option where you can run anything that you'd like. In this way we reduce the risk of breaking anyone downstream when we make any overhauls to the base image or reorder where the commands are run.

Reply:
That's true, but many of users aren't super familiar with Docker - this is also the reason we exposed `files` and `python_dependencies` as keywords, to help optionally customize the image without building a base image yourself.  But the risk of breakage is a good point, happy to leave this as-is.

Reply:
I think I'll leave it as is for now, it doesn't prevent us from exposing this in the future.